src/VecSim/algorithms/brute_force/brute_force.cpp

```diff
@@ -49,6 +49,10 @@ size_t BruteForceIndex::estimateInitialSize(const BFParams *params) {
     return est;
 }
 
+size_t BruteForceIndex::estimateElementMemory(const BFParams *params) {
+    return params->dim * sizeof(float);
```

Comment:
Actually, why not consider the meta-data as well? size of `VectorBlockMember`, and `labelToIdLookup`entry?